molecule/default/tests/test_default.py

```diff
@@ -52,6 +85,24 @@ def test_services(host, svc):
 
 
 @pytest.mark.parametrize("command", ["docker-compose version"])
-def test_command(host, command):
+def test_commands(host, command):
     """Test that appropriate commands are available."""
+    distribution = host.system_info.distribution
+    codename = host.system_info.codename
+
+    if distribution in ["debian"]:
+        if codename in ["buster", "bullseye"]:
+            assert all([host.run(cmd).rc == 0 for cmd in ["docker compose version"]])
+        elif codename in ["stretch", "bookworm"]:
+            assert all([host.run(cmd).rc == 0 for cmd in ["docker-compose version"]])
+        else:
+            assert False, f"Unknown codename {codename}"
+    elif distribution in ["kali"]:
+        assert all([host.run(cmd).rc == 0 for cmd in ["docker-compose version"]])
+    elif distribution in ["ubuntu"]:
+        assert all([host.run(cmd).rc == 0 for cmd in ["docker compose version"]])
+    elif distribution in ["amzn", "fedora"]:
+        assert all([host.run(cmd).rc == 0 for cmd in ["docker-compose version"]])
+    else:
+        assert False, f"Unknown distribution {distribution}"
     assert host.run(command).rc == 0
```

Comment:
```suggestion
```

Reply:
Already taken care of when I force-pushed the latest commit.